src/Build.OM.UnitTests/Definition/ProjectCollection_Tests.cs

```diff
@@ -1269,7 +1266,6 @@ public void ProjectCollectionChangedEvent2()
             project.SetProperty("a", "1");
             collection.SetGlobalProperty("a", "1");
             VerifyProjectCollectionEvents(collection, true, "2");
-            project.GetPropertyValue("a").Equals("2", StringComparison.OrdinalIgnoreCase);
```

Comment:
Rather than delete this change to `project.GetPropertyValue("a").ShouldBe("2")`? I suppose it could be testing that it didn't throw, but more likely a mstest -> nunit -> xunit conversion issue.

Reply:
I found after enabling this and the others like it in Assert.IsTrue() lines that they were in fact wrong - because there is no full path on the project XML provided to the ProjCollection, the project is not added to the collection, and so changes to the collection global properties were not propagating. After debugging for awhile I figured I'd dump the lines instead of asserting false, since they're useless.

LMK if you disagree, I can restore under a false assertion.

Reply:
That sounds concerning. Should we fix the full-path thing instead?

Reply:
The doc comment on ProjectCollection.OnAfterRenameLoadedProject() says no:

        /// Stores the project in the list of loaded projects if it has a name.
        /// Does not store the project if it has no name because it has not been saved to disk yet.
        /// If the project previously had a name, but was not in the collection already, throws InvalidOperationException.
        /// If the project was not previously in the collection, sets the collection's global properties on it.